### ISSUE TITLE
What is the APIs to test the motors when mocking pitch/roll/yaw+throttle 

### ISSUE THREAD

**Issue by weixingsun** [betaflight/betaflight-configurator#1117]
Dear Experts,

I am building my own copter, but I cannot find any document about testing the motors without a remote control, please let me know where is the codes to control the Throttle + Pitch/Roll/Yaw
Or give me a hint to develop it , thanks

**Comment by mikeller** [comment]
Open configurator, click 'Connect', click 'Motors' tab, click switch next to 'I understand the risks', slide 'Master' slider up, watch motors turn.

**Comment by weixingsun** [comment]
@mikeller thanks for your reply, I know this page, and am testing motors as you mentioned, 
it looks good, but I am finding the "combined" movements like yaw/pitch/roll, just like what we do from a remote control.

**Comment by weixingsun** [comment]
Still wondering if there is a way to mocking the remote control signals

**Comment by McGiverGim** [comment]
In the receiver tab, there is an MSP virtual command, I don't know if it will work... you can try it but with caution.

**Comment by weixingsun** [comment]
@McGiverGim Thanks for your reply, I didn't find the MSP virtual command feature in the receiver tab,
there is a page shows the R/P/Y/T and AUX bars but without input fields, please help

**Comment by McGiverGim** [comment]
I don't remember if they move with keys or with the mouse. I think remember they are keys (shortcuts). But you must first agree to the message that appears.

**Comment by weixingsun** [comment]
![betaflight-receiver](https://user-images.githubusercontent.com/5229930/42896505-acd8c3e8-8aef-11e8-9921-90d99b881525.png)
This is my screenshot, I tried key AERT on my keyboard, no luck. this should be protocol of different RC not the keyboard.

**Comment by McGiverGim** [comment]
You must activate the MSP RX Receiver in the configuration tab. Then you will see a new button in the bottom right of this screen. Or something similar.

**Comment by weixingsun** [comment]
@McGiverGim  Yes, I followed your step and open the control stick page, I can control the AUX signals now. 
But the Roll/Pitch/Yaw/Throttle are still unable to operate on my Macbook, I will try windows version later. 
Thanks a lot

**Comment by weixingsun** [comment]
I can use the motor tab to control the speed, but I cannot change my motors from the control stick page under receiver tab, the only effect is the preview animation in the bottom of the page. 
Do I need to enable any configuration?

**Comment by McGiverGim** [comment]
I'm not too sure... maybe in the new versions you can't mount the FC while connected to the configurator. Try to go to the motors tab and activate the arming and then try again.

**Comment by mikeller** [comment]
@weixingsun: You need to enable motor control in the Motors tab in order for arming to be possible.

**Comment by weixingsun** [comment]
OK, but for whatever reason, my motors only run under motors tab, no luck under receiver tab. 
My FC: Omnibus F4v3
Betaflight-Configurator: 10.3.1 mac
anything I can do ?

**Comment by mikeller** [comment]
Looks like your RX isn't working properly. `MSP` will go away when you enable the motors in the motor tab, and `CLI` is shown because you are in CLI.

**Comment by weixingsun** [comment]
@mikeller I upgraded FC to the latest version 3.4.0, then I see the value changed to "RXLOSS CLI"
without MSP, but I still cannot give it a run

**Comment by mikeller** [comment]
@weixingsun: Again, your RX is not working. Set up a working RX and it will work.

**Comment by McGiverGim** [comment]
If I'm not wrong, you must open the MSP RX input panel, that is a standalone window, and then you can change to the motors tab, activate it, and use it.
